src/components/views/rooms/MemberTile.tsx

```diff
@@ -143,32 +155,32 @@ export default class MemberTile extends React.Component {
         });
     }
 
-    getStatusMessage() {
+    private getStatusMessage(): string {
         const { user } = this.props.member;
         if (!user) {
             return "";
         }
-        return user._unstable_statusMessage;
+        return user.unstable_statusMessage;
```

Comment:
whoops

Reply:
[This seems alright](https://github.com/matrix-org/matrix-js-sdk/blob/develop/src/models/user.ts#L46)?

Reply:
Yup, whoops that I left it broken and for you to fix :P

Reply:
Ah! :smile: